src/antsibull_docs/lint_plugin_docs.py

```diff
@@ -311,6 +311,82 @@ def _create_lookup(
     return lookup
 
 
+class _ValidationWalker(dom.NoopWalker):
+    def __init__(self, markup_validator: "_MarkupValidator", key: str):
+        self.markup_validator = markup_validator
+        self.errors = markup_validator.errors
+        self.key = key
+
+    def process_error(self, part: dom.ErrorPart) -> None:
+        self.errors.append(f"{self.key}: Markup error: {part.message}")
+
+    def process_bold(self, part: dom.BoldPart) -> None:
+        if not part.text:
```

Comment:
Would it be a good idea to call `strip()` for each of the checks for empty markup?

`if not part.text.strip():`

Reply:
Generally non-empty values that are only whitespace can be ok, especially for `C()` and `V()`. I only used `strip()` where whitespace only makes no sense, i.e. for URLs and RST references.

Reply:
makes sense. thanks for explaining!